packages/react-data-grid-addons/src/draggable/DraggableHeaderCell.js

```diff
@@ -1,40 +1,30 @@
 import { DragSource } from 'react-dnd';
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { DragItemTypes } from 'common/constants';
 
 class DraggableHeaderCell extends Component {
-
   componentDidMount() {
-    let connectDragPreview = this.props.connectDragPreview;
-    let img = new Image();
+    const { connectDragPreview } = this.props;
+    const img = new Image();
     img.src = './assets/images/drag_column_full.png';
-    img.onload = function() {
-      connectDragPreview(img);
-    };
-  }
-
-  setScrollLeft(scrollLeft) {
```

Comment:
This is no longer required as `HeaderCell` content is wrapped inside a `DraggableHeaderCell`